functions/Install-DbaFirstResponderKit.ps1

```diff
@@ -196,7 +196,7 @@ function Install-DbaFirstResponderKit {
             $allprocedures_query = "select name from sys.procedures where is_ms_shipped = 0"
             $allprocedures = ($server.Query($allprocedures_query, $Database)).Name
             # Install/Update each FRK stored procedure
-            foreach ($script in (Get-ChildItem $LocalCachedCopy -Recurse -Filter "sp_*.sql")) {
+            foreach ($script in (Get-ChildItem $LocalCachedCopy -Recurse -Filter "sp_*.sql" -Exclude '*SQL_Server_2005.sql')) {
```

Comment:
It will be safer and more maintainable to exclude the directory the files are in. Additional files can be put in the deprecated folder at their discretion, using this method we are only ignoring the 2005 scripts.
```suggestion
            foreach ($script in (Get-ChildItem $LocalCachedCopy -Recurse -Filter "sp_*.sql" -Exclude 'Deprecated')) {
```

Reply:
Sorry, but that doesn't work.

Reply:
Currently all script are in the root of the repo. I've checked them back to to the tags of 2016, they were always in the root folder. I think we can use this, the two `Get-ChildItem` are required to skip the root folder:
```
Get-ChildItem $LocalCachedCopy | Get-ChildItem -Filter "sp_*.sql"
```

Reply:
I'm good with that...

Reply:
@andreasjordan if you can try this with just:

```
foreach ($script in (Get-ChildItem $LocalCachedCopy -Filter "sp_*.sql")) {
```

Reply:
@wsmelton 
@claudiospizzi

Ok, I've done some testing:
$LocalCachedCopy is 'C:\Users\Administrator\AppData\Roaming\PowerShell\dbatools\FRK_main'
The sp_Bitz-Script is at 'C:\Users\Administrator\AppData\Roaming\PowerShell\dbatools\FRK_main\SQL-Server-First-Responder-Kit-main\SQL-Server-First-Responder-Kit-main\sp_Blitz.sql'
So there are two folders between them.
To get it to work without -Recurse, here is the correct line of code:
`            foreach ($script in (Get-ChildItem $LocalCachedCopy | Get-ChildItem | Get-ChildItem -Filter "sp_*.sql")) {
`
Not very elegant, but that would be another possibility to fix this with only changing one line.

To be a little more direct: I don't care how you fix this, but please fix it. Soon. The commandlet is just not working correct at the moment.

Reply:
@andreasjordan  you are more than welcome to have the fix in your fork of the module. I'm not looking to band-aid commands only to have to go back and work them a second or third time.